packages/graph/hash_graph/lib/graph/src/types/schema/object.rs

```diff
@@ -20,14 +21,17 @@ struct ObjectRepr<V> {
     required: Vec<BaseUri>,
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(try_from = "ObjectRepr<V>", rename_all = "camelCase")]
+pub trait ValidateUri {
+    fn validate_uri(&self, base_uri: &BaseUri) -> Result<(), ValidationError>;
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize)]
```

Comment:
This line was removed here. Was that intentional?
`#[serde(try_from = "ObjectRepr<V>", rename_all = "camelCase")]`